### ISSUE TITLE
Crash Server goaloptimizer-1.16.5-1.1.0-hotfix

### ISSUE THREAD

**Issue by Aarods** [MCTeamPotato/GoalOptimizer#4]
![image](https://github.com/MCTeamPotato/GoalOptimizer/assets/110215990/8dae2c75-ecf5-4337-856c-a0fabb9b3744)

GoalOptimizer in the latest version crashes when entering in certain specific chunks or directly at startup (Server side as far as I tested).
[crash-2023-07-04_12.51.16-server.txt](https://github.com/MCTeamPotato/GoalOptimizer/files/11949497/crash-2023-07-04_12.51.16-server.txt)

**Comment by Kasualix** [comment]
I'll inject into Enhanced AI to fix this.
This is the same modification as Vampirism, which also causes a performance issue.

**Comment by Kasualix** [comment]
[Unzip this.zip](https://github.com/MCTeamPotato/GoalOptimizer/files/11958771/Unzip.this.zip)
Can you try this build? Summon some creepers to test.

**Comment by Aarods** [comment]
![image](https://github.com/MCTeamPotato/GoalOptimizer/assets/110215990/4756a4de-59ca-4c3a-8c83-9fe22363d19f)
I already tried, the server crashes again.
[crash-2023-07-05_14.23.21-server.txt](https://github.com/MCTeamPotato/GoalOptimizer/files/11958921/crash-2023-07-05_14.23.21-server.txt)

**Comment by Aarods** [comment]
Edit 1: Sorry, I copied an old version by accident. Now it lets you enter and when spawning a creeper it doesn't crash (only with GoalOptimizer), when testing it with RoadRunner the game crashes again (GoalOptimizer with RoadRunner), with Redirectionor it works fine as far as I saw.
Crash with RoadRunner: [crash-2023-07-05_15.13.06-server.txt](https://github.com/MCTeamPotato/GoalOptimizer/files/11959514/crash-2023-07-05_15.13.06-server.txt)

Edit 2: ![image](https://github.com/MCTeamPotato/GoalOptimizer/assets/110215990/36b205f7-4a72-4805-b473-7c4ad9ecb4b7)
After testing for a while and turning on and off, the server crashes automatically when a player logs in.
[crash-2023-07-05_15.21.17-server.txt](https://github.com/MCTeamPotato/GoalOptimizer/files/11959552/crash-2023-07-05_15.21.17-server.txt)

Edit 3: When I enter the world and move around for a while there is another crash, I don't know if it is related to the creepers, but when I move away a little bit it closes automatically with another error. (I'm not sure if its the same crash as the other one)
[crash-2023-07-05_15.30.27-server.txt](https://github.com/MCTeamPotato/GoalOptimizer/files/11959640/crash-2023-07-05_15.30.27-server.txt)
